apps/sandbox/src/pages/_app.tsx

```diff
@@ -1,6 +1,37 @@
-import '@/styles/globals.css'
-import type { AppProps } from 'next/app'
+import '@bos-web-engine/ui/reset.css';
+import '@bos-web-engine/ui/styles.css';
+import '@bos-web-engine/wallet-selector-control/styles.css';
+import '@bos-web-engine/sandbox/styles.css';
+import '@near-wallet-selector/modal-ui/styles.css';
+import '@/styles/globals.css';
+
+import type { AppProps } from 'next/app';
+import s from '@/styles/app.module.css';
+import Link from 'next/link';
+import { NearIconSvg, Theme } from '@bos-web-engine/ui';
+import { WalletSelectorControl } from '@bos-web-engine/wallet-selector-control';
+import { useWallet } from '@/hooks/useWallet';
```

Comment:
I'd prefer we stick to the convention of non-aliased import paths for the sake of consistency with the rest of the codebase. Not a blocker for this PR but maybe a good candidate for a follow up PR that adds the project-wide lint configuration to this package.

Reply:
Sure thing, I'll remove the alias import. An ES Lint rule would probably be a good idea in the future to prevent this. I believe VS Code autocompleted the import with the alias.

Reply:
Thanks for the input! I definitely appreciate things being consistent too. I happen to find Next JS path aliases really convenient when working on deeply nested pages. For example:

```ts
// before
import { Button } from '../../../../components/button'
 
// after
import { Button } from '@/components/button'
```

My preference would be to use the default path alias Next JS provides whenever you're in the context of a Next JS app. Whenever you're not inside a Next JS app, we wouldn't use aliases.

If you and others feel strongly though, I'm happy to disable the default alias for consistency and move on. 👍

Reply:
+1 for the path alias in Next.js projects, crawling up the directory tree with `..` is frustrating

Consistency is fair, but could we see this as the first step towards progressively using the alias across `apps/web` and `apps/demo`? Or would you rather not move towards the alias in the long run @andy-haynes ?

Reply:
Thanks for the clarification, I didn't realize that was a full on NextJS convention. I've only ever seen aliasing abused in really old Node projects, but that makes sense for an opinionated modern framework. Definitely leave it as is.

Reply:
Missed your comment @mpeterdev - definitely in favor of applying this to `apps/web` as well. Anything to bring these projects adhering to NextJS conventions 🙂

Reply:
@andy-haynes I just pushed a commit that brings consistency to `app/web` and `apps/sandbox` - they both share the same TS config and default alias now (and both have a `src` directory).